system-test/logging.js

```diff
@@ -14,20 +14,15 @@
  * limitations under the License.
  */
 
-'use strict';
```

Comment:
Is this intentional? What is the rationale?

Reply:
I am going to run around and do this in all of our TypeScript libs, but we don't need this in our source files.  With the `strict` TypeScript compiler option set, we add `use strict` to all output files by default:
https://github.com/google/ts-style/blob/master/tsconfig-google.json#L14